src/content/en/2025/cookies.md

```diff
@@ -3,18 +3,646 @@
+
+{{ figure_markup(
+  image="top-third-party-cookies-set.png",
+  caption="Top third-party cookies and domains that set them.",
+  description="The chart shows the most widely-set third-party cookies. DoubleClick sets third-party advertising cookies on a little over 35% of pages. Microsoft also sets advertising cookies on 23% of pages. All top 10 domains setting third-party cookies are related to tracking and advertising.",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vSzdHAn-vwJ-Z05NYWZrImgKaX0q5D-jgWay8FD9lMDj2jr3cEjozE083JOSi6cZZX37vVD2TjEKw28/pubchart?oid=232078905&format=interactive",
+  sheets_gid="503090386",
+  sql_file="top_20_third_party_cookies.sql"
+  )
+}}
+
+<!-- markdownlint-disable-next-line MD051 -->
+Similarly, [Figure 13](#fig-13) shows the top 10 most common third-party cookies being created on the top 1M websites. The `IDE` and `test_cookie` cookies are set by `doubleclick.net` (owned by Google) and are present on more than 35% and 25% of websites. DoubleClick checks if a user's web browser supports third-party cookies by trying to set `test_cookie`. `MUID` from Microsoft comes next, present on more than 23% of websites, and is also used for targeted advertising and cross-site tracking. As already pointed out in the [`Partitioned` cookies](#partitioned-chips-proposal) section, this year we do not observe anymore the `YSC` and `VISITOR_INFO1_LIVE` from YouTube among top third-party cookies.
+
+{# TODO would be nice to investigate a little more what is behind youtube not using these anymore and maybe be able to say more here #}
+
+{{ figure_markup(
+  image="top-cookie-domains.png",
+  caption="Top registrable domains setting cookies.",
+  description="The chart shows the most common domains that set cookies on the web. Google's owned advertising platform DoubleClick sets cookies on more than 33% of the top 1M websites while others in this top 10 domains are at about 5% to 15%.",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vSzdHAn-vwJ-Z05NYWZrImgKaX0q5D-jgWay8FD9lMDj2jr3cEjozE083JOSi6cZZX37vVD2TjEKw28/pubchart?oid=483296297&format=interactive",
+  sheets_gid="503090386",
+  sql_file="top_20_domains_setting_cookies.sql"
+  )
+}}
+
+<!-- markdownlint-disable-next-line MD051 -->
+Perhaps, unsurprisingly from prior results, the 10 most common domains ([Figure 14](#fig-14)) that set cookies on the web are all involved with search, targeting, and advertising services. Google's combined coverage (`doubleclick.net`, `google.com`, and `youtube.com`) is reaching more than 50% of the websites, and Microsoft's (`bing.com`, `clarity.ms`, `linkedin.com`) 30%.
```

Comment:
1. "Google's combined coverage (`doubleclick.net`, `google.com`, and `youtube.com`) is reaching more than 50% of the websites" is this true? The query result is not backing this up, as the sites that use google.com or youtube.com could be a subset of doubleclick.net. Is there a different query that looks at the combined unique sites of doubleclick, google and youtube?